src/SFA.DAS.ProviderApprenticeshipsService.Web/Controllers/HomeController.cs

```diff
@@ -55,5 +64,50 @@ public IActionResult CookieDetails()
         {
             return View();
         }
+
+        [HttpGet]
+        [Authorize]
+        [AllowAllRoles]
+        [Route("~/signin", Name = RouteNames.SignIn)]
+        public async Task<IActionResult> SignIn()
```

Comment:
The reason why I have moved this action method from Account Controller to Home Controller because the Account Ctrl has got the base controller with Policies enabled but we need to the Roles Mapping(SaveIdentityAttributes()) once the Signin is completed.
This happens as part of SecurityTokenValidated() in case of IdAMS. But for DfESignIn since we are using Nuget Package this can't be overridden.